src/Sushi.php

```diff
@@ -2,90 +2,91 @@
 
 namespace Sushi;
 
+use DateTime;
 use Illuminate\Database\Connectors\ConnectionFactory;
-use Illuminate\Support\Str;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\SQLiteConnection;
+use Illuminate\Support\Arr;
+use Illuminate\Support\LazyCollection;
 
+/** @mixin Model */
 trait Sushi
 {
-    protected static $sushiConnection;
+    protected static SQLiteConnection $sushiConnection;
```

Comment:
Don't enforce PHP 7.4 please, or else this cannot even be installed on Laravel 5.8, with current PHP version constraints :-(

Reply:
Is Laravel 5.8 PHP7.4 incompatible? If so, ***I** will remove it from the possible versions. If it's moved into this package is up to @calebporzio

Reply:
Laravel 5.8 is not incompatible with PHP 7.4; Laravel PHP version requirement is 7.1.3

Reply:
https://github.com/laravel/framework/blob/5.8/.travis.yml
Laravel 5.8 introduced a test runner for PHP 7.4 - so there's no general issue with this combination.